app/assets/stylesheets/views/_covid.scss

```diff
@@ -10,6 +10,16 @@ $covid-grey: #272828;
   }
 }
 
+.covid__page-title {
+  @include govuk-font(48, $weight: bold);
+  margin-bottom: govuk-spacing(5);
+
+  @include govuk-media-query($from: tablet) {
+    margin-top: govuk-spacing(4);
+    margin-bottom: 45px;
```

Comment:
could this be replaced by `govuk-spacing(7) + 5` or just `govuk-spacing(7)` or `govuk-spacing(8)`?

It will help keep the spacing consistent